simple_gait_controller_node:
  ros__parameters:
    use_sim_time: true
    min_vel_x: -0.4
    min_vel_y: -0.2
    max_vel_x: 0.4
    max_vel_y: 0.2
    max_vel_theta: 1.0
    stance_duty: 0.50
    stance_height: 0.01
    swing_height: 0.04
    walking_height: 0.20
    controller_hz: 200.0
    base_frame_id: base_link
    joint_control_topic: position_controller/commands
    joint_names:
      - lf_hip_joint
      - lf_upper_leg_joint
      - lf_lower_leg_joint
      - rf_hip_joint
      - rf_upper_leg_joint
      - rf_lower_leg_joint
      - lb_hip_joint
      - lb_upper_leg_joint
      - lb_lower_leg_joint
      - rb_hip_joint
      - rb_upper_leg_joint
      - rb_lower_leg_joint

teleop_twist_joy_node:
  ros__parameters:
    use_sim_time: true
    axis_linear:  # Left thumb stick vertical & horizontal
      x: 1
      y: 0
    scale_linear:
      x: 0.25
      y: 0.25

    axis_angular:  # Right thumb stick rotate
      yaw: 3
    scale_angular:
      yaw: 0.4

    enable_button: 5  # RB button
